src/main/java/Calculator/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package Calculator;
+
+import java.awt.desktop.OpenFilesEvent;
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String expression = scan.nextLine();
+
+        String[] values = expression.split(" ");
+        System.out.println(calculate(values));
+        scan.close();
+    }
+
+    public static int calculate(String[] values) {
+        Operator operator;
+        int nextValue;
+        int result = Integer.parseInt(values[0]);
+        for (int i = 1; i < values.length; i += 2) {
+            nextValue = Integer.parseInt(values[i + 1]);
+            operator = Operator.findOperator(values[i]);
+            result = operator.operate(result, nextValue);
```

Comment:
```suggestion
            result = Operator.findOperator(values[i])
	            .operate(result, nextValue);
```

Operator.findOperator()가 정적 메서드로 정의되어 있기 때문에 별도로 Operator를 위에서 선언한 뒤에 사용할 필요가 없어 보이네요.!
메서드 체이닝이라는 방법으로 연결해서 사용하면 좋을 것 같네요.!